app.css

```diff
@@ -158,3 +159,56 @@ a{
     height: 10rem;
     border-radius: 100%;
 }
+
+/* alerts */
+.alert-div{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+}
+.alert{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 0px solid black;
```

Comment:
you can use border:none instead of this !!

Reply:
Done Thanks